src/components/LearnerActivityTable/index.jsx

```diff
@@ -48,12 +48,11 @@ const InactivePastMonthLearnersTable = () => {
     },
   ];
 
-
-  const formatPassedTimestamp = (timestamp) => {
-    if (timestamp) {
-      return formatTimestamp({ timestamp });
+  const getTableColumns = props => {
```

Comment:
You don't actually need to pass in `props` to this function as you should already have them in the outer scope ([the props passed into the component itself](https://github.com/edx/edx-portal/pull/74/commits/aede8b733ad14b747f807f7502718eaced71b69a#diff-9510153a701fb5abeb9028052670ee72R7)).